app/Services/SubscriberService.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+
+namespace App\Services;
+
+
+use App\Subscriber;
+use App\Tools\ResponseTool;
+
+class SubscriberService
+{
+    public static function subscribe($email)
+    {
+        $checkEmail = Subscriber::whereEmail($email)->count();
+        if ($checkEmail == 0) {
```

Comment:
No need to store it as variable, make it through
```suggestion
        if (Subscriber::whereEmail($email)->count()) {
```